generator/ServiceClientGeneratorLib/Generators/Endpoints/EndpointResolver.tt

```diff
@@ -96,6 +96,10 @@ namespace <#=Config.Namespace#>.Internal
             }
 
 <#}#>
+<#
```

Comment:
I wouldn't mention a potential V5. What about: `GetACL and PutACL are deprecated in V4 and may be removed in the future`?